go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/integr8ly/integreatly-operator
 go 1.13
 
 require (
-	github.com/3scale/3scale-operator v0.4.0
+	github.com/3scale/3scale-operator v0.2.1-0.20200327122530-d376df6cc71f
```

Comment:
Curious why not use the `v0.5.0` tag here? https://github.com/3scale/3scale-operator/releases/tag/v0.5.0

Reply:
Oh didn't realise they've tagged a 0.5.0 release now. They didn't have that tag when I was testing the update. I'll update the PR to use that tag now :+1: